--- ./src/bin/systemd-crontab-generator.py.old	2021-03-07 10:34:43.362030743 +0100
+++ ./src/bin/systemd-crontab-generator.py	2021-03-07 10:35:06.289909296 +0100
@@ -475,8 +475,7 @@
         f.write('ExecStart=%s\n' % command)
         if job['e']:
              f.write('Environment=%s\n' % job['e'])
-        if job['u'] != 'root':
-             f.write('User=%s\n' % job['u'])
+        f.write('User=%s\n' % job['u'])
         if standardoutput:
              f.write('StandardOutput=%s\n' % standardoutput)
         if 'Z' in job and job['Z']:
