diff --git a/mollysocket-vapid.service b/mollysocket-vapid.service
index 07830e7..de59118 100644
--- a/mollysocket-vapid.service
+++ b/mollysocket-vapid.service
@@ -3,7 +3,7 @@ Description=Generate MollySocket VAPID key
 
 [Service]
 Type=oneshot
-ExecStart=sh -c '( [ -f $CONFIGURATION_DIRECTORY/conf.toml ] || echo "db = \'$STATE_DIRECTORY/ms.db\'" > $CONFIGURATION_DIRECTORY/conf.toml ); ( [ -f $CONFIGURATION_DIRECTORY/vapid.key ] || ( ms vapid gen | systemd-creds encrypt - $CONFIGURATION_DIRECTORY/vapid.key ) )'
+ExecStart=sh -c '( [ -f $CONFIGURATION_DIRECTORY/conf.toml ] || echo "db = \'$STATE_DIRECTORY/mollysocket.db\'" > $CONFIGURATION_DIRECTORY/conf.toml ); ( [ -f $CONFIGURATION_DIRECTORY/vapid.key ] || ( mollysocket vapid gen | systemd-creds encrypt - $CONFIGURATION_DIRECTORY/vapid.key ) )'
 RemainAfterExit=true
 
 ConfigurationDirectory=mollysocket
diff --git a/mollysocket.service b/mollysocket.service
index 3f85dd6..d603e75 100644
--- a/mollysocket.service
+++ b/mollysocket.service
@@ -24,7 +24,7 @@ UMask=0007
 ProtectHome=true
 ProtectSystem=true
 
-ExecStart=ms server
+ExecStart=mollysocket server
 KillSignal=SIGINT
 
 Restart=on-failure
