--- a/src/tracks/labeltrack/ui/LabelGlyphHandle.cpp	2023-02-21 11:44:47.725028378 +0100
+++ b/src/tracks/labeltrack/ui/LabelGlyphHandle.cpp	2023-02-21 11:44:58.205006340 +0100
@@ -21,7 +21,7 @@
 #include "ViewInfo.h"
 #include "../../../SelectionState.h"
 #include "../../../ProjectAudioIO.h"
-#include "../../../images/Cursors.h"
+#include "../../../../images/Cursors.h"
 #include "../../../tracks/ui/TimeShiftHandle.h"
 
 #include <wx/cursor.h>
--- a/src/tracks/playabletrack/wavetrack/ui/WaveClipTrimHandle.cpp	2023-05-15 12:32:44.579941831 +0200
+++ b/src/tracks/playabletrack/wavetrack/ui/WaveClipTrimHandle.cpp	2023-05-15 12:32:55.555077003 +0200
@@ -16,7 +16,7 @@
 #include "../../../../TrackArtist.h"
 #include "../../../../Snap.h"
 #include "../../../../TrackPanelDrawingContext.h"
-#include "../../../images/Cursors.h"
+#include "../../../../../images/Cursors.h"
 #include "WaveClip.h"
 #include "WaveTrack.h"
 #include "WaveTrackView.h"
