--- a/src/CMakeLists.txt	2023-02-21 14:40:56.655263342 +0100
+++ b/src/CMakeLists.txt	2023-02-21 14:41:10.119229084 +0100
@@ -1351,15 +1351,10 @@
                FILES_MATCHING PATTERN "*.so*" )
       install( FILES "${_INTDIR}/${APP_NAME}.desktop"
                DESTINATION "${_DATADIR}/applications" )
-      install( FILES "${topdir}/LICENSE.txt" "${topdir}/README.md"
-               DESTINATION "${_DATADIR}/doc/${APP_NAME}" )
       install( FILES "${_SRCDIR}/audacity.xml"
                DESTINATION "${_DATADIR}/mime/packages" )
       install( FILES "${topdir}/presets/EQDefaultCurves.xml"
                DESTINATION "${_PKGDATA}" )
-      install( PROGRAMS "${PROJECT_SOURCE_DIR}/linux/tenacity.sh"
-               DESTINATION "."
-               RENAME "tenacity" )
    endif()
 endif()
 
