--- src/CMakeLists.txt.orig	2023-02-21 13:10:04.523277632 +0100
+++ src/CMakeLists.txt	2023-02-21 13:11:08.060172027 +0100
@@ -1357,9 +1357,6 @@
                DESTINATION "${_DATADIR}/mime/packages" )
       install( FILES "${topdir}/presets/EQDefaultCurves.xml"
                DESTINATION "${_PKGDATA}" )
-      install( PROGRAMS "${PROJECT_SOURCE_DIR}/linux/audacity.sh"
-               DESTINATION "."
-               RENAME "audacity" )
    endif()
 endif()
 
