--- geeqie-9999/configure.in.old	2017-12-04 08:59:45.408878000 +0100
+++ geeqie-9999/configure.in	2017-12-04 08:59:55.743889462 +0100
@@ -17,9 +17,6 @@
 AC_PREREQ(2.57)
 AC_INIT([geeqie], m4_esyscmd_s(git rev-parse --quiet --verify --short HEAD), [geeqie-devel@lists.sourceforge.net], [], [http://www.geeqie.org/])
 
-# Add -Werror to the default CFLAGS
-CFLAGS+=" -Werror -Wno-error=deprecated-declarations -Wno-error=sign-compare -Wno-error=return-type"
-
 # Check for rightly dirs
 AC_CONFIG_SRCDIR([src/main.c])
 
